src/player.js

```diff
@@ -226,6 +226,19 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _streamType: string;
+  /**
+   * The current playback attributes state
+   * @type {Object}
+   * @private
+   */
+  _playbackAttributesState: { [attribute: string]: any } = {
+    MUTED: undefined,
```

Comment:
@yairans why use CAPITAL? it's not a const just an object map. please use lower case.

Reply:
done